faros_event.sh

```diff
@@ -318,6 +323,9 @@ function processEventTypes() {
         addArtifactToData
         addCommitToData
         addRunToData
```

Comment:
We might want to separate the input validation from the event construction logic now that we are also using it for the CE path.

Reply:
Ack.